src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+@import 'header';
+@import 'variables';
```

Comment:
It's a good practice to import variables first since they might be used in other style files. Thus, the 'variables' import should come before any other imports.